### DISCUSSION TITLE
Use Tailwind insted of PrimeFlex

### DISCUSSION THREAD

**Discussion by decodeitorg** [primefaces/community#2180] [PrimeReact] [answered]
Why don't we just use Tailwind instead of PrimeFlex? It reduces the complexity of pass-through, results in a smaller bundle size, and improves design consistency.

**Comment by melloware** [comment, marked as answer]
PrimeTek is working on PrimeReact Tailwind to do exactly that and its just like PrimeVue Tailwind: https://tailwind.primevue.org/

**Reply by decodeitorg** [reply]
wow, that exactly what i am looking for.
By the way, can you please teel, what is the current situation of development? how many days might it need to complete?

**Reply by melloware** [reply]
I don't work for PrimeTek so I am not sure of their internal timelines

**Reply by gumush** [reply]
Any updates ?

**Reply by melloware** [reply]
@gumush still in progress...

**Reply by dfaivre-pcs** [reply]
any tracking issue we can subscribe to?

**Reply by melloware** [reply]
> any tracking issue we can subscribe to?

Not that I am aware of this work is happening internal at PrimeTek on their private repo.

**Reply by melloware** [reply]
I have updated all PrimeReact Starters to use Tailwind4 in styled and unstyled mode: https://github.com/primefaces/primereact-examples
